### PR TITLE
User job list

### PR DESCRIPTION
**Pull request by ExerciseAndrew**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ app.get('/data/employers', employers.retrieveEmployerData);
 
 app.post('/appliedJobs', applications.oneClickApply);
 
-app.get('/appliedJobs', applications.getAppliedJobs);
+app.get('/appliedJx`obs', applications.getAppliedJobs);
```
Comment (Gobleizer) [inline server/index.js:74]: Is this a regex for something?

File: src/app/components/UsersJobList/Main.jsx [hunk 2]
```diff
@@ -2,27 +2,39 @@ import * as React from 'react';
...
     return (
-      <div>Visit the job search page to save and apply for jobs!</div>
-    );
-  }
-
-  return (
     <Container sx={{ display: 'flex' }}>
       <Box>
-        <UsersJobsList listOfJobs={jobsState} interestLevel={interestLevel} />
+        <UserJobsList listOfJobs={savedJobsList} interestLevel={interestLevel} />
```
Comment (Gobleizer) [inline src/app/components/UsersJobList/Main.jsx:33]: Good refactor here.

File: src/app/pages/Jobs.jsx [hunk 3]
```diff
@@ -4,97 +4,103 @@ import {
...
+        getSavedJobs(buttonName);
         break;
-      case 'VERY INTERESTED':
-        getSavedJobs(interestLevel);
+      case 'Very Interested':
+        getSavedJobs(buttonName);
         break;
-      case 'INTERESTED':
-        getSavedJobs(interestLevel);
+      case 'Interested':
```
Comment (Gobleizer) [inline src/app/pages/Jobs.jsx:57]: Good change for consistency, thank you.

**Review by Gobleizer — APPROVED** [review]
Great updates based on the feedback, and you've resolved more issues too. Thank you.

**Comment by Gobleizer** [comment]
@maki-q Please do a review and merge if you approve.

**Review by maki-q — APPROVED** [review]
LGTM

**Pull request merged**